source/config.cpp

```diff
@@ -350,6 +356,20 @@ bool Config::is_buffer_protocol_requested(string const &class__) const
 	return false;
 }
 
+bool Config::is_module_local_requested(string const &namespace_) const
+{
+	const string namespace_all = "ALL";
+	auto module_local_all = std::find(module_local_namespaces.begin(), module_local_namespaces.end(), namespace_all);
```

Comment:
Can we shortcut this a bit? and add `if( module_local_all != module_local_namespaces.end()) return true;` That way we will not need to run second search at all.